src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -54,12 +54,15 @@ public enum ApiIds
         public ApiEvent(
             ICoreLogger logger,
             ICryptographyManager cryptographyManager,
-            string correlationId) : base(EventNamePrefix + "api_event", correlationId)
+            Guid correlationId) : base(EventNamePrefix + "api_event")
```

Comment:
This is great, string operations take a lot of CPU time and this is on the hot path.